addons/web/static/src/legacy/js/fields/special_fields.js

```diff
@@ -93,7 +93,12 @@ var FieldTimezoneMismatch = FieldSelection.extend({
 
         if (this.mismatch){
             $span.insertAfter(this.$el);
-            $span.attr('title', _t("Timezone Mismatch : This timezone is different from that of your browser.\nPlease, set the same timezone as your browser's to avoid time discrepancies in your system."));
```

Comment:
in js: `this.nodeOptions.title`

Reply:
also, not sure about the name of the option. This title is only displayed when there's a mismatch, so maybe `mismatch_title` ?

Reply:
Hello @jdoutreloux,
Could you have a look at this and implement what Aaron suggested?

Thanks 🙂

Reply:
Hi @IT-Ideas it's already done. I can't change the name of the span attribute but I changed the name of the option to mismatch_title as suggested.